proto/sge/sportevent/params.proto

```diff
@@ -38,16 +38,4 @@ message Params {
     (gogoproto.nullable)   = false
   ];
 
-  string event_max_vig = 5 [
```

Comment:
event_max_loss : This field is not used anywhere. Please remove it.

Reply:
done